app/controllers/overrides/sessions_controller.rb

```diff
@@ -0,0 +1,4 @@
+class Overrides::SessionsController < DeviseTokenAuth::SessionsController
```

Comment:
@flamesoft that's most likely because the user details you provided are not correct :wink: -- I just tested on my computer with your last changes and it's all good.